omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -121,22 +121,35 @@
 
   # Install the checks along with their dependencies
   block do
+    if windows?
+      wheel_build_dir = "#{windows_safe_path(project_dir)}\\.wheels"
+      build_deps_dir = "#{windows_safe_path(project_dir)}\\.build_deps"
+    else
+      wheel_build_dir = "#{project_dir}/.wheels"
+      build_deps_dir = "#{project_dir}/.build_deps"
+    end
+
     #
     # Prepare the build env, these dependencies are only needed to build and
     # install the core integrations.
     #
-    command "#{pip} install hatchling==0.11.2"
+    command "#{pip} download --dest #{build_deps_dir} hatchling==0.11.2"
+    command "#{pip} download --dest #{build_deps_dir} setuptools==40.9.0" # Version from ./setuptools2.rb
```

Comment:
Why change the directory where these are installed?

Reply:
No change but rather introduced so `pip` can see them in the temporary environment it creates

Reply:
Ah ok got it, Thanks!